server/routes/bookedtoday/arrivals/state.ts

```diff
@@ -66,6 +80,6 @@ const SearchDetailsCodec: Codec<SearchDetails> = {
 }
 
 export const State = {
-  searchDetails: stateOperations('search-details', SearchDetailsCodec),
-  newArrival: stateOperations('new-arrival', NewArrivalCodec),
+  searchDetails: new StateOperations('search-details', SearchDetailsCodec),
+  newArrival: new StateOperations('new-arrival', NewArrivalCodec),
```

Comment:
Now a class so can more easily access type info compared to an anonymous function